eventsourcing/src/test/java/org/axonframework/eventsourcing/AnnotationBasedEventStateApplierTest.java

```diff
@@ -184,7 +183,7 @@ void invokesOnlyMostSpecificHandler() {
         }
     }
 
-    // feel free to drop the support in case of some changed in EventHandling logic (like Empty stream always returned from event handling component)
+    @Disabled("TODO #XXXX - Event Sourcing Handler support for evolved state return")
```

Comment:
Is there already an issue number for this? ;-)

Reply:
I left XXXX, to do the issue :D

Reply:
Done: https://github.com/AxonFramework/AxonFramework/issues/3313